code/addons/a11y/src/preview.test.tsx

```diff
@@ -76,6 +76,7 @@ const violations = [
 
 describe('afterEach', () => {
   beforeEach(() => {
+    vi.clearAllMocks();
```

Comment:
This is no longer standard?
Should we set it up in some config file?, because this is my personal assumption that tests always get run with a cleared mock.

Reply:
This was added by Valentin to fix a flake.